platform/ios/platform/ios/scripts/deploy-swift-package.sh

```diff
@@ -106,11 +106,17 @@ make xcframework
 step "Zipping xcframeworks…"
 MAPBOX_ZIP_FILE="Mapbox-${PUBLISH_VERSION}.zip"
 (cd ${BINARY_DIRECTORY} && rm -f ${MAPBOX_ZIP_FILE} && zip -yr ${MAPBOX_ZIP_FILE} Mapbox.xcframework)
+METAL_ANGLE_ZIP_FILE="MetalANGLE-${PUBLISH_VERSION}.zip"
+(cd ${BINARY_DIRECTORY} && rm -f ${METAL_ANGLE_ZIP_FILE} && zip -yr ${METAL_ANGLE_ZIP_FILE} MetalANGLE.xcframework)
```

Comment:
The version numbers of the binaries are as below, but I think this script pulls the `{PUBLISH_VERSION}` from the git tags.  If we cannot tag the MetalANGLE, then we may be able to use the binaries version as it is on disk.

![image](https://user-images.githubusercontent.com/118112/112561445-77d32500-8d92-11eb-80b7-c8e1a65adb28.png)